### PR TITLE
Bookstore: styling

### PR DESCRIPTION
**Pull request by Gh-Reza**
## Changes:

- Added styles using CSS3
- Added styles using Bootstrap

### CODE REVIEW

**Review by tugsem — CHANGES_REQUESTED** [review]
##  STATUS: REQUIRED CHANGES ♻️
Hello @Gh-Reza 👋 , 

Good job so far! 👏 
There are some issues that you still need to work on to go to the next project but you are almost there!

Alt Text
## Highlights 🚀 
✔️code is clean
✔️style almost identical
## Required Changes ♻️
- [ ] Please make these titles uppercased, remove the underlines, and color accordingly to match the zeplin design:

your design|zeplin
---|---
image|image

- [ ] Please use the same icon to match the zeplin design: 

your design|zeplin
---|---
image|image


- [ ] Please use the same value for the colors and font weights rather than using Bootstrap 'primary' to match  the design:


your design|zeplin
---|---
image|image


- [ ] Please add the vertical short lines to match the zeplin design:

your design|zeplin
---|---
image|image

- [ ] Please use the same color and letter case for this title to match the designs

your design|zeplin
---|---
image|image


_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Book.jsx [hunk 1]
```diff
@@ -44,9 +44,9 @@ const Book = ({
...
     </div>
     <div className="col d-flex justify-content-center align-items-start flex-column">
-      <p className="mb-0 text-uppercase completed">Current Chapter</p>
+      <p className="mb-0 text-uppercase completed current-chapter">Current Chapter</p>
       <p className="chapter">{chapter}</p>
-      <button type="button" className="btn btn-primary px-4">Update prograss</button>
+      <button type="button" className="btn btn-primary px-4 text-uppercase">Update prograss</button>
     </div>
   </div>
 );
```
Comment (Tufahel) [inline src/components/Book.jsx:52]: - [ ] Kindly check the color of the fonts from the template and fix them according to the previous reviewer.
image

**Review by Tufahel — CHANGES_REQUESTED** [review]
# Status: Required Changes :recycle:
Hi @Gh-Reza :wave:  

Good job so far! 👍 
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights 🌟

Alt Text

✔️ Linters checks are passing.
✔️ Git flow is correct.
✔️ Readme is professional.
✔️ App is working fine.

## Required Changes ♻️

***Please kindly check the comments/review below.***

## Optional suggestions 🤷‍♂️

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers, and Happy coding!👏👏👏

## Important Notes 📝
- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.
- You can mail me at [Mail](mailto:tufahel96@gmail.com) or ask any question at Slack, `Name: Tufahel`

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by tufoinnkuo10 — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi @Gh-Reza 

Your project is complete! Great job for naking the changes requested 🥇. There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉💯🌟 

## Good Points 👍
- Descriptive Pull Request
- Works as expected
- great design
- No linter errors

<br>
Cheers..Happy coding!..and keep soaring higher! 💻 🍷 🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**